### ISSUE TITLE
Enhancement: XML Modal select all

### ISSUE THREAD

**Issue by evanlaferriere** [WheatonCS/Lexos#366]
Given that people will be working with XML documents with many, many tags, and probably want similar thing to happen to most of the tags, it might be useful to enable a dropdown that sets the action for every tag. I'm unsure how to go about doing this

**Comment by scottkleinman** [comment]
Just to clarify, do you mean another dropdown at the top of the list with something like "Apply to all tags"?

**Comment by evanlaferriere** [comment]
yes

**Comment by scottkleinman** [comment]
I started working on this and then realised the following:

1. The default setting strips all tags.
2. Applying "Replace Element and All Its Contents" to every element will delete the contents of the entire file.
3. It is very unlikely that the user will want to replace every element with an attribute value (at least, in the context of literary text analysis).

So that means that an "Apply to all tags" option isn't really useful. What I think we need to do instead is make clear that leaving the defaults will strip all tags.

**Comment by mleblanc321** [comment]
>make clear that leaving the defaults will strip all tags.
yes, evan, let's drop a note/tooltip/pop-up to say this;  

scott:  evan is working on the regex to pull off the  "Replace Element and All Its Contents"  functionalities; more soon

m

**Comment by scottkleinman** [comment]
I'm working on the UI end. I'll push in a few minutes.

**Comment by scottkleinman** [comment]
OK, I've pushed the changes. These are as follows:

1. The "XML Handling" button has been renamed "Scrub Tags".
2. The "Strip Tags" check box has been commented out. Users can use "Scrub Tags" with the default "Remove Tag Only" setting for each tag to achieve the same effect.
3. New text has been added explaining that the functions handle XML, HTML, and SGML tags and to highlight that the default settings strip all tags.
4. "Save Changes" (now on the left) triggers a processing icon briefly to tell the user that something has happened.
